src/PresentationalComponents/PathwaysPanel/PathwaysPanel.js

```diff
@@ -18,17 +18,31 @@ import propTypes from 'prop-types';
 import { useGetPathwaysQuery } from '../../Services/Pathways';
 import { useIntl } from 'react-intl';
 import { useSelector } from 'react-redux';
+import { workloadQueryBuilder } from '../Common/Tables';
 
 const PathwaysPanel = () => {
   const intl = useIntl();
   const { offset } = useSelector(({ filters: { pathState } }) => pathState);
   const [expanded, setExpanded] = useState(
     JSON.parse(localStorage.getItem('advisor_pathwayspanel_expanded') || 'true')
   );
+  const selectedTags = useSelector(({ filters }) => filters.selectedTags);
+  const workloads = useSelector(({ filters }) => filters.workloads);
+  const SID = useSelector(({ filters }) => filters.SID);
+  let options = {};
+  selectedTags?.length &&
+    (options = {
+      ...options,
+      ...{ tags: selectedTags.join(',') },
+    });
+  workloads &&
+    (options = { ...options, ...workloadQueryBuilder(workloads, SID) });
```

Comment:
This could also be written as 

```js
  const options = {
    ...selectedTags?.length > 0 ? { tags: selectedTags.join(',') } : {},
    ...workloads ? workloadQueryBuilder(workloads, SID) : {}
  }
```

or?

Reply:
ahh I like that better, sending it